pulp_2to3_migration/app/models/content.py

```diff
@@ -33,9 +33,10 @@ class Pulp2Content(BaseModel):
     downloaded = models.BooleanField(default=False)
     pulp3_content = models.ForeignKey(Content, on_delete=models.SET_NULL, null=True)
     pulp2_repo = models.ForeignKey(Pulp2Repository, on_delete=models.SET_NULL, null=True)
+    pulp2_subid = models.CharField(max_length=255, blank=True)
```

Comment:
:tada:

Reply:
Question: Should I send something on the mailing list that `blank=True` is preferable to `null=True` for string fields? I am pretty sure there are a lot of `null=True` string fields around the various Pulp repositories. How concerned should we be about this?

Reply:
I think it's always good to have such conversations on pulp-dev. I thought we had similar in the past but if we did it was looong ago. +1 to write to the mailing list and see why we didn't go that path in some cases.

Reply:
Done: https://www.redhat.com/archives/pulp-dev/2021-January/msg00049.html